### PR TITLE
Display Bob version in settings tab and "About" panel

### PR DESCRIPTION
**Pull request by pinheadmz**
Closes https://github.com/kyokan/bob-wallet/issues/112

Also updated a bit of the menu syntax.

Screen Shot 2020-04-10 at 4 17 41 PM

### CODE REVIEW

File: app/menu.js [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+    // iconPath: Linux & Windows only
+    app.setAboutPanelOptions({
+      applicationName: pkg.productName, 
+      applicationVersion: pkg.version,
+      copyright: `License: ${pkg.license}`,
+      version: pkg.version,
+      credits: pkg.author,
+      author: pkg.author,
+      website: pkg.homepage,
+      iconPath: Path.join(__dirname, '../resources/icons/128x128.png')
```
Comment (pinheadmz) [inline app/menu.js:23]: @mslipper this may need to be tested on linux / windows specifically

**Comment by mslipper** [comment]
Dig it!

File: app/menu.js (hunk 1 above)
Comment (mslipper) [reply app/menu.js:23]: I think you should use this guy, coupled with `path.join`: https://www.electronjs.org/docs/api/process#processresourcespath-readonly

File: app/menu.js [hunk 2]
```diff
@@ -39,22 +59,22 @@ export default class MenuBuilder {
...
+          role: 'about'
         },
         {type: 'separator'},
         {label: 'Services', submenu: []},
         {type: 'separator'},
         {
           label: 'Hide Bob',
           accelerator: 'Command+H',
-          selector: 'hide:'
+          role: 'hide'
```
Comment (mslipper) [inline app/menu.js:70]: Are the colons you removed here causing issues? Curious what these changes fix.

Comment (pinheadmz) [reply app/menu.js:70]: We can remove this commit if you like. What happened was, I was reviewing the existing menu code and couldn't find documentation for the `selector` property. Then I found this: https://github.com/electron/electron/pull/3554 indicating that `selector` was deprecated and replaced by `role`. The available values for `role` are documented here: https://www.electronjs.org/docs/all#roles

Comment (mslipper) [reply app/menu.js:70]: Nope, works for me. Glad this has been updated.

File: app/menu.js (hunk 1 above)
Comment (pinheadmz) [reply app/menu.js]: Fixed in ffd8470d8e27ab7079aade315aeb0e7354166b4b

This works in the MacOS build (although I think it is actually ignored by MacOS, see https://www.electronjs.org/docs/api/app#appsetaboutpaneloptionsoptions)

I spent a day trying and failing to get Windows installed on a partition on my MacBook Pro so I still can't currently test on Windows. Can you check this works? I'll acquire some dedicated hardware for Window soon.

Comment (mslipper) [reply app/menu.js]: I'll check this out when we build the next release. Merging for now.

**Pull request merged**